src/pudl/output/ferc1.py

```diff
@@ -2043,27 +2051,27 @@ def in_explosion_tables(table_name: str, in_explosion_table_names: list[str]) ->
 class XbrlCalculationForestFerc1(BaseModel):
     """A class for manipulating groups of hierarchically nested XBRL calculations.
 
-    We expect that the facts reported in high-level FERC tables like
+    We expect that the facts reported in less granular FERC tables like
     :ref:`core_ferc1__yearly_income_statements_sched114` and
     :ref:`core_ferc1__yearly_balance_sheet_assets_sched110` should be
     calculable from many individually reported granular values, based on the
     calculations encoded in the XBRL Metadata, and that these relationships should have
-    a hierarchical tree structure. Several individual values from the higher level
+    a hierarchical tree structure. Several individual values from the less granular
     tables will appear as root nodes at the top of each hierarchy, and the leaves in
```

Comment:
maybe say "at the base of each hierarchy" instead of "top" so it's more "rooty"

Reply:
The word "several" and "appear" here are confusing. Which roots we use is intentional, right?
```
Several individual values from the less granular
tables will appear as root nodes at the top of each hierarchy
```

Reply:
yea i went back and forth on this one. i think "top of hierarchy" is more analogous to "root of tree" than "base of hierarchy". bc base of a hierarchy implies that its a wide (i.e. granular) base. and the top of a hierarchy is usually one of more single points (see) from which the rest of the hierarchy flows down (towards leaves)

Reply:
if you keep reading below here you'll see that a list of seed nodes can be given to this forest builder, which is the way we intentionally say what root(s) to start building from

Reply:
I think that's why I was confused about the word "appear" in the section I highlighted. The language portrays something that happens coincidentally when in reality it is planned. But this is too nuanced spend time on rly.

Reply:
they do appear as roots until we "prune" all of the nodes that are not connected to the seed node (this stuff is a little weedsy and is documented in the many class methods